src/core/backend.h

```diff
@@ -43,7 +43,7 @@ namespace nvidia { namespace inferenceserver {
         if (response != nullptr) {                                        \
           LOG_STATUS_ERROR(                                               \
               InferenceResponse::SendWithStatus(                          \
-                  std::move(response), status__),                         \
+                  std::move(response), (S)),                              \
```

Comment:
Don't want to evaluate `S` twice. Will declaring `status__` to `const auto` solve the issue?

Reply:
Right. We pass the function call in S as well. Let me see.

Reply:
Done